lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -68,8 +76,13 @@ class PlatformViewManager {
   /// `factoryFunction` needs to be a [PlatformViewFactory].
   bool registerFactory(String viewType, Function factoryFunction,
       {bool isVisible = true}) {
-    assert(factoryFunction is PlatformViewFactory ||
-        factoryFunction is ParameterizedPlatformViewFactory);
+    assert(
```

Comment:
I think you should revert these changes so this PR doesn't conflict with https://github.com/flutter/engine/pull/42255

Reply:
Good catch! I'll just move this change to the other PR.